### ISSUE TITLE
Book custom gui ? 1.14

### ISSUE THREAD

**Issue by LittleFoe** [sp614x/optifine#3526]
i'm trying to match the ui of custom books match the cit look. But, the custom gui doc only reffers to containers, is it not possible to do this? If not how does one make a custom book skin to explain a datapack nicely?

**Comment by realjackasterisk** [comment]
I don't believe this is possible yet, would be nice though.

**Comment by LittleFoe** [comment]
Yes it would. I don't understand why it's not implemented, it seems like it would use the same code that's used for containers. The only thing i can think of is that it's not in high demand from the community, cause modders can implement it them self of course if they need a custom book.
But now that datapacks are gaining traction i think it's time to have this feature.

**Comment by StrangeChestnut** [comment]
YES I WANT IT
